tensorflow/python/ops/script_ops.py

```diff
@@ -641,7 +641,7 @@ def py_func(func, inp, Tout, stateful=True, name=None):
 
 @tf_export("numpy_function")
 @dispatch.add_dispatch_support
-def numpy_function(func, inp, Tout, name=None):
+def numpy_function(func, inp, Tout, stateful=True, name=None):
```

Comment:
Sorry, missed this bit - please make sure the flag defaults to False. Otherwise it breaks backward compatibility.

Reply:
Nevermind, I see the old code defaulted to True as well.